language/move-prover/bytecode-to-boogie/test_mvir/test-specs-verify.mvir

```diff
@@ -24,23 +22,25 @@ module TestSpecs {
       if (exists<R>(get_txn_sender())) {
         abort(1);
       }
-      move_to_sender<R>(R{x:1});
```

Comment:
Was looking at this locally and wondering why it was identical to the previous test!

Reply:
Shows us that the lightweight diagnosis verification has its draw backs... but it appears we can't do easily better, because boogie line numbers are often wrong.